src/platform/test/functional/apps/discover/group2_data_grid2/_data_grid_new_line.ts

```diff
@@ -110,7 +110,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       cell = await dataGrid.getCellElementExcludingControlColumns(0, 0);
       content = await cell.findByCssSelector('.unifiedDataTable__cellValue');
       expect(await content.getVisibleText()).to.be(VALUE_WITHOUT_NEW_LINES);
-      expect(await content.getComputedStyle('white-space')).to.be('nowrap');
```

Comment:
Can you please look into why this change happened? Seems unrelated to the issue the PR is solving.

Reply:
This is intentional change and enforced by this line: https://github.com/elastic/kibana/pull/238874/files#diff-53c4ac17fb33a558f816de16f98917dba20b625e319b53c163b83fd1e7bb623dR1508
The reason is a line clamp, which needs `white-space: normal` in order to work correctly.
But I'm open to a different approach as well!